services/src/main/java/org/keycloak/protocol/oidc/endpoints/LogoutEndpoint.java

```diff
@@ -308,6 +308,8 @@ private Response displayLogoutConfirmationScreen(LoginFormsProvider loginForm, A
     @NoCache
     @Consumes(MediaType.APPLICATION_FORM_URLENCODED)
     public Response logout() {
+        Cors.builder().allowAllOrigins().add();
```

Comment:
Perhaps we want to allow only the origins configured with the client by looking at the ID Token or the client_id parameter?

Reply:
+1 , we know the client at this point https://github.com/keycloak/keycloak/blob/main/services/src/main/java/org/keycloak/protocol/oidc/endpoints/LogoutEndpoint.java#L213 , so maybe at this point we can set origins based on that?